py/kubeflow/tests/prow_artifacts_test.py

```diff
@@ -112,26 +112,26 @@ def testCreateSymlink(self): # pylint: disable=no-self-use
         "gs://some-bucket/pr-logs/pull/fake_org_fake_name/72"
         "/kubeflow-presubmit/100")
 
-  @mock.patch("kubeflow.testing.test_util.get_num_failures")
-  @mock.patch("kubeflow.testing.prow_artifacts._get_actual_junit_files")
-  def testCheckNoErrorsSuccess(self, mock_get_junit, mock_get_failures):
-    # Verify that check no errors returns true when there are no errors
-    gcs_client = mock.MagicMock(spec=storage.Client)
-    artifacts_dir = "gs://some_dir"
-    mock_get_junit.return_value = set(["junit_1.xml"])
-    mock_get_failures.return_value = 0
-    self.assertTrue(prow_artifacts.check_no_errors(gcs_client, artifacts_dir))
-
-  @mock.patch("kubeflow.testing.test_util.get_num_failures")
-  @mock.patch("kubeflow.testing.prow_artifacts._get_actual_junit_files")
-  def testCheckNoErrorsFailure(self, mock_get_junit, mock_get_failures):
-    # Verify that check no errors returns false when a junit
-    # file reports an error.
-    gcs_client = mock.MagicMock(spec=storage.Client)
-    artifacts_dir = "gs://some_dir"
-    mock_get_junit.return_value = set(["junit_1.xml"])
-    mock_get_failures.return_value = 1
-    self.assertFalse(prow_artifacts.check_no_errors(gcs_client, artifacts_dir))
+  # @mock.patch("kubeflow.testing.test_util.get_num_failures")
```

Comment:
Either delete it or mark it as expected failure but don't comment it out.
https://docs.pytest.org/en/latest/skipping.html

I think its fine to delete it since the ksonnet code is dead code and we should get rid of it.